packages/components/src/utils/table/table.ts

```diff
@@ -0,0 +1,49 @@
+import type { Row, Table } from '@tanstack/react-table';
+
+export const selectRow = <TData>(table: Table<TData>, rowId?: string) => {
+  if (!rowId || rowId === '') {
+    table.setRowSelection({});
+  } else {
+    table.setRowSelection({ [`${rowId}`]: true });
```

Comment:
why not simply use { [rowId]: true }?

Reply:
I would make a singleliner out of this, and because of this I'm not sure about the real utilize of this function. Is it really needed to have a simplification of something which is already simple?

`table.setRowSelection(rowId === undefined || row === '' ? {} : {[rowId]: true})`

Reply:
I like the current implementation better because it is much easier to read.
I also like that this is a defined function, again because of readability when this is needed. This function is currently called 4 different times in the Variables Editor.